cms/envs/common.py

```diff
@@ -1366,6 +1366,8 @@
     }
 }
 
+STUDIO_FRONTEND_CONTAINER_URL = None
```

Comment:
This may be more correctly located in https://github.com/edx/edx-platform/blob/master/cms/envs/devstack_docker.py

Reply:
Well, this value is going to be checked even in production so I need to initialize it to something for all environments. I'm going to add some instructions in the studio-frontend README about this, but the idea is that by default Studio will load SFE files from node_modules. But, if you set this value to `http://localhost:18011` in your `private.py` then it will instead load the files from your studio-frontend docker container.